internal/kafka/internal/api/public/api/openapi.yaml

```diff
@@ -652,6 +652,78 @@ paths:
       security:
       - Bearer: []
       summary: Returns the list of supported regions of the supported cloud provider
+  /api/kafkas_mgmt/v1/instance_types/{cloud_id}/{cloud_region}:
+    get:
+      operationId: getInstanceTypesByCloudProviderAndRegion
+      parameters:
+      - description: ID of the supported cloud provider
+        explode: false
+        in: path
+        name: cloud_id
+        required: true
+        schema:
+          type: string
+        style: simple
+      - description: Name of the supported cloud provider region
+        explode: false
+        in: path
+        name: cloud_region
+        required: true
+        schema:
+          type: string
+        style: simple
+      - description: Page index
```

Comment:
Do we need pagination for this? I think many integrations will rely on complete list

Reply:
Thanks @pb82 @wtrocki I'll remove pagination